test/service/test_drs.py

```diff
@@ -140,46 +141,64 @@ def test_drs(self):
                         # We expect a DRS object with an access URL
                         expected['access_methods'] = [
                             {
-                                'access_url': {
-                                    'url': ('https://org-hca-dss-checkout-prod.s3.amazonaws.com/'
-                                            'blobs/307.a72.eb6?foo=bar&et=cetera')
-                                },
+                                'access_url': {'url': 'https://org-hca-dss-checkout-prod.s3.amazonaws.com/'
+                                                      'blobs/307.a72.eb6?foo=bar&et=cetera'},
                                 'type': 'https'
```

Comment:
DOesn't this correlate with the enum value?

Reply:
Yes it does. Are you asking me to change something here?

Reply:
In standup we decided to leave the assertion strings as they were.